runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpPanel.java

```diff
@@ -53,6 +54,7 @@
 @Slf4j
 class XpPanel extends PluginPanel
 {
+	@Getter
```

Comment:
Plz don't

This is already messy as is, don't hook into other classes internal state so directly.